packages/perseus/src/widgets/interactive-graphs/mafs-styles.css

```diff
@@ -214,53 +216,22 @@
 .MafsView pattern g {
     stroke: rgba(33, 36, 44, 0.32);
 }
-.axis-tick-labels {
-    font-size: 14px;
-    font-family: "Mafs-MJXTEX";
-    line-height: 1.5em;
-    /* Prevent labels from being selected, and from interfering with click and
-     * drag interactions on the graph. */
-    user-select: none;
-    pointer-events: none;
-    /* Prevent labels from covering up interaction points. */
-    z-index: -1;
-}
-
-.y-axis-tick-labels {
-    width: 1.75em;
-    display: flex;
-    flex-flow: column;
-    transform: translateX(calc(-100% - 0.5em));
-    position: absolute;
-    text-align: right;
-}
-.y-axis-right-of-grid {
-    transform: translateX(calc(50% + 0.5em));
-}
-.x-axis-tick-labels {
-    display: flex;
-    flex-flow: row;
-    flex-direction: row-reverse;
-    position: absolute;
-    transform: translateY(calc(50% - 0.25em));
-}
-.x-axis-top-of-grid {
-    transform: translateY(calc(-100% + 0.25em));
-}
-
-.y-axis-tick-labels span {
-    display: inline-block;
-    height: var(--y-axis-label-height, 20px);
-}
-
-.x-axis-tick-labels span {
-    text-align: center;
-    width: var(--x-axis-label-width, 20px);
-}
 
 .MafsView .angle-arc,
 .MafsView .arc-right-angle {
     stroke: var(--mafs-blue);
-
     stroke-width: 0.1px;
 }
+
+.MafsView .axis-tick-label {
+    /* There is a bug in the MathJax font for Safari, so we use KaTeX */
+    font-family: "KaTeX_Main";
```

Comment:
Could we use mathjax for all fonts EXCEPT the one media query with safari to use KaTeX